### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ostUp**
[DEMO LINK](https://ostUp.github.io/react_movies-list-filter/)

### CODE REVIEW

**Review by Roman-Artiukh — APPROVED** [review]
GJ

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    return movie.title.toLowerCase().includes(query.toLowerCase())
```
Comment (AndriiRut) [inline src/App.tsx:10]: you can create a function to avoid duplicating your code

**Review by AndriiRut — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const filmTextCheck = (text: string, searchQuery: string) => (
+    text.toLowerCase().includes(searchQuery)
+  );
```
Comment (AndriiRut) [inline src/App.tsx:11]: wtite this function before component and component's props type

**Review by AndriiRut — APPROVED** [review]
🚀

**Pull request closed without merging**